lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,559 @@
+---
+name: Parameters
+field_name_base_url: http://hl7.org/fhir/r4/parameters-definitions.html#Parameters
+fields:
+
+  - name: resourceType
+    required: 'Yes'
+    type: string
+    description: The type of the FHIR resource.
+    action: modify
+    example: |
+      {
+        "resourceType": "Parameters"
+      }
+    note: resourceType must be ChargeItem
```

Comment:
This should be 'Parameters'.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/2e95e8f7f7c9760a3c5d4d4f5df05fb503cb5543#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R15